src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,214 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 58 to help you plan.
+
+{# wf_updated_on: 2017-03-16 #}
+{# wf_published_on: 2017-03-16 #}
+{# wf_tags: deprecations,removals,chrome58 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 58 to help you plan. #}
+
+# Deprecations and Removals in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 58,
+which is in beta as of March 16. This list is subject to change at any time.
+
+
+## Remove RTCRtcpMuxPolicy of "negotiate"
```

Comment:
The entire `rtcpMuxPolicy` setting will be removed, as otherwise it would just be an enum with a single value. @taylor-b to confirm. (I can't find zhihuang on GitHub.)